src/sqlitestorage.cpp

```diff
@@ -198,63 +198,69 @@ bool SqliteStorage::open()
   sqlite3_busy_timeout( d->mDatabase, 1500 );
 
   /* Create Calendars, Components, etc. tables */
-  query = CREATE_VERSION;
-  sqlite3_exec( d->mDatabase );
+#ifdef Q_OS_UNIX
+  if (d->mSem.isInitialProcess()) {
```

Comment:
Well, you only want one process to create the database tables, right?  If you have a schema version check or something, then it might be ok, but there's no need for other processes to consider creating/updating/integrity checking the DB.